Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -79,7 +80,7 @@
                                                     infoClass="text_blue" showDetail="true" showSummary="true"
                                                     tooltip="true"/>
 
-                                        <!-- Box für die Bearbeitung der Details -->
+                                        <!-- Box f�r die Bearbeitung der Details -->
```

Comment:
Yes. GitHub is too smart when showing that diff - it shows the same umlaut, no matter whether it is encoded in ISO-8859-1 or UTF-8.